src/main/java/com/rackspace/ceres/app/services/MetadataService.java

```diff
@@ -213,7 +223,36 @@ public MetadataService(ReactiveCqlTemplate cqlTemplate,
         )
         .flatMapMany(Flux::fromIterable);
   }
+  
+  public Flux<TsdbQuery> getMetricsAndTagsAndMetadata(List<TsdbQueryRequest> queries, List<Granularity> granularities) {
+    List<TsdbQuery> result = new ArrayList<TsdbQuery>();
+    
+    for (TsdbQueryRequest query : queries) {
+      List<Map<String, String>> filters = query.getFilters();
+      
+      for (Map<String, String> filter : filters) {
+        String[] splitValues = filter.get("filter").split("\\|");
```

Comment:
Why do we need to split on | ?

Reply:
We only support "type", "literal_or" but you have a point maybe I need to enforce it?

Reply:
I think splitting would be required in case of literal_or filter as there can be `|` separated values that we need to or.